examples/MQ/histogramServer/FairMQExHistoDevice.cxx

```diff
@@ -0,0 +1,76 @@
+/********************************************************************************
+ *    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+#include <chrono>
+#include <thread>
+
+#include "FairMQExHistoDevice.h"
+#include "RootSerializer.h"
+
+#include "TH1F.h"
+#include "TH2F.h"
+#include "TRandom3.h"
+#include "TObjArray.h"
+#include "TMessage.h"
+#include "TMath.h"
+
+FairMQExHistoDevice::FairMQExHistoDevice()
+    : FairMQDevice()
+    , fRandom(new TRandom3(0))
+    , fArrayHisto(new TObjArray())
+{
+}
+
+FairMQExHistoDevice::~FairMQExHistoDevice() {}
+
+void FairMQExHistoDevice::InitTask()
+{
+    fh_histo1 = new TH1F("histo1", "", 200, -0.5, 1.5);
```

Comment:
Can all fh_histos be value members instead of pointers?
TH1F seems to have `operator=` defined, so instead of putting them on the heap here, we can do:
```C++
fh_histo1 = TH1F("histo1", "", 200, -0.5, 1.5);
```